posthog/api/integration.py

```diff
@@ -127,7 +127,8 @@ def authorize(self, request: Request, *args: Any, **kwargs: Any) -> HttpResponse
                 raise ValidationError("Kind not configured")
         elif kind == "github":
             query_params = urlencode({"state": token})
-            installation_url = f"https://github.com/apps/{'posthog-error-tracking'}/installations/new?{query_params}"
+            # TODO: change this to env variable
+            installation_url = f"https://github.com/apps/{'posthog-local-dev'}/installations/new?{query_params}"
```

Comment:
This we need to do before merging

Reply:
Probably quicker than adding an ENV var is to check if you're running in dev / prod mode